src/client/conn/http2.rs

```diff
@@ -60,13 +64,13 @@ pub struct Builder {
 pub async fn handshake<E, T, B>(
     exec: E,
     io: T,
-) -> crate::Result<(SendRequest<B>, Connection<T, B>)>
+) -> crate::Result<(SendRequest<B>, Connection<T, B, E>)>
 where
-    E: Executor<BoxSendFuture> + Send + Sync + 'static,
-    T: AsyncRead + AsyncWrite + Unpin + Send + 'static,
+    T: AsyncRead + AsyncWrite + Unpin + 'static,
     B: Body + 'static,
     B::Data: Send,
-    B::Error: Into<Box<dyn StdError + Send + Sync>>,
+    B::Error: std::error::Error + Send + Sync + 'static,
```

Comment:
Can this change be undone?

Reply:
I changed all `Error` bounds. Now the test works again with `Box<dyn Error ...>`